testapi.pm

```diff
@@ -1982,6 +1983,20 @@ sub eject_cd {
     query_isotovideo(backend_eject_cd => \%nargs);
 }
 
+=head2 toggle_network
+
+  toggle_network network_enabled => $boolean;
+
+Changes the default qanet0 device's state, fatal if fails.
+
+=cut
+
+sub toggle_network {
```

Comment:
as you add a new function to the test API please bump the API version in OpenQA/Isotovideo/Interface.pm